Jellyfin.Api/Controllers/PlaystateController.cs

```diff
@@ -91,18 +101,27 @@ public class PlaystateController : BaseJellyfinApiController
         /// <param name="userId">User id.</param>
         /// <param name="itemId">Item id.</param>
         /// <response code="200">Item marked as unplayed.</response>
-        /// <returns>A <see cref="OkResult"/> containing the <see cref="UserItemDataDto"/>.</returns>
+        /// <response code="400">Item not found.</response>
```

Comment:
```suggestion
        /// <response code="404">Item not found.</response>
```

Reply:
Doh! Silly me, that's what I get for copying the line above 😅 Fixed!